Source/FakeItEasy/Core/FakeWrapperConfigurator.cs

```diff
@@ -47,6 +47,13 @@ public IFakeOptionsBuilder<T> WithAdditionalAttributes(IEnumerable<CustomAttribu
 
         public IFakeOptionsBuilder<T> Implements(Type interfaceType)
         {
+            Guard.AgainstNull(interfaceType, "interfaceType");
+
+            if (!interfaceType.IsInterface)
+            {
+                throw new ArgumentException("The specified type is not an interface");

```

Comment:
Thanks for that, my Markdown leaves much to be desired!  As mentioned above, I've included the ```Type.Name``` in the exception message now, so for the failing tests it reads *"The specified type 'Foo' is not an interface"*.